src/dex/balancer-v2/balancer-v2-math.ts

```diff
@@ -1,5 +1,24 @@
-export const BZERO = BigInt(0);
-export const BONE = BigInt(1);
+import {
```

Comment:
When we get to such long list of imported symbols, we may just as much do `import * as BIs from '../../bigint-constants'` and then use them with the namespace such as `BIs.BI_0`

Reply:
Thanks, that is a good idea. Currently we don't need it anymore, but in future I will memorize this approach :)